pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -176,6 +186,23 @@ func (c *clusterAutoscaler) Deploy(ctx context.Context) error {
 		return err
 	}
 
+	if c.HasExpanderConfigMapConfigured() {
+		configMap := &corev1.ConfigMap{}
+		if err := c.gardenClient.Get(ctx, client.ObjectKey{Name: *c.config.ExpanderConfig.ConfigMapName, Namespace: c.projectNamespace}, configMap); err != nil {
+			return err
+		}
+
+		configMap.Namespace = c.namespace
+		configMap.Name = ClusterAutoScalerPriorityConfigMapName
+
+		if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, c.client, configMap, func() error {
```

Comment:
This is the seed client, but the `ConfigMap` lives in the `kube-system` of the Shoot.

Reply:
The namespace comes from a defaulted CLI flag: https://github.com/gardener/autoscaler/blob/45e190f4bd7890e0029bef8eb9affa0946f135d5/cluster-autoscaler/main.go#L116
We don't override it, so CA looks for the `ConfigMap` in the `kube-system` namespace of the shoot cluster. This also matches what I have observed in real systems.

Reply:
Ah, alright! Thank you very much :)